exercises/standard_library_types/iterators5.rs

```diff
@@ -0,0 +1,113 @@
+// iterators5.rs
+
+// Rustling progress is modelled using a hash map. The name of the exercise is
+// the key and the progress is the value. Two counting functions were created
+// to count the number of exercises with a given progress. These counting
+// functions use imperative style for loops. Recreate this counting
+// functionality using iterators.
+// Execute `rustlings hint iterators5` for hints.
+//
+// Make the code compile and the tests pass.
+
+// I AM NOT DONE
+
+use std::collections::HashMap;
+
+#[derive(PartialEq, Eq)]
+enum Progress {
+    None,
+    Some,
+    Complete,
+}
+
+fn count_for(map: &HashMap<String, Progress>, value: Progress) -> usize {
+    let mut count = 0;
+    for val in map.values() {
+        if val == &value {
+            count += 1;
+        }
+    }
+    count
+}
+
+fn count(map: &HashMap<String, Progress>, value: Progress) -> usize {
```

Comment:
I'd rename these to `count_iterator` or something, to keep them separate from the `_for` methods.